src/core/app/init.ts

```diff
@@ -34,17 +39,32 @@ export async function appInit() {
     appConfigResponse.user = defaultUserProfileData;
   }
 
-  setAppLanguage(appConfigResponse.user.language);
+  const language = appConfigResponse.user.language;
+  setAppLanguage(language);
+
+  const defaultLayers = await getDefaultLayers(initialState.app, language);
+
+  const { basemapLayerId, basemapLayerUrl } = getBasemapFromDetails(defaultLayers);
 
   if (initialState.layers === undefined) {
-    initialState.layers = await getDefaultLayers(initialState.app);
+    initialState.layers = defaultLayers?.map((l) => l.id) ?? [];
   } else {
     // HACK: Remove KLA__ prefix from layers ids coming from url
     initialState.layers = initialState.layers.map((l) => l.replace(/^KLA__/, ''));
+
+    const basemapInUrl = findBasemapInLayersList(initialState.layers);
+    if (!basemapInUrl) {
+      initialState.layers.push(basemapLayerId);
+    }
```

Comment:
This will enable basemap after page refresh event if user disable it. If it intended - better to describe this in comment. Also, looks like we forget to check case when default base map not listed app config (in case we want map without basemap in some app)
```suggestion
    /* Enable app default basemap layer if in not listed in url */
    const basemapInUrl = findBasemapInLayersList(initialState.layers);
    if (!basemapInUrl) {
      initialState.layers.push(basemapLayerId);
    }
```